scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -317,15 +317,18 @@ private static void RemovePdbsFromDir(string directory)
             FS.RmFilesInDirRecursive(directory, "*.pdb");
         }
 
-        private static void CopySharedHost(string outputDir)
+        private static void CopySharedHost(BuildTargetContext c, string outputDir)
         {
             // corehost will be renamed to dotnet at some point and then this can be removed.
             File.Copy(
                 Path.Combine(Dirs.Corehost, CoreHostBaseName),
                 Path.Combine(outputDir, $"dotnet{Constants.ExeSuffix}"), true);
+            
+            string versionedFxrDir = Path.Combine(outputDir, "host", "fxr", c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion);
```

Comment:
Can we pass in the versionedFxrDir instead of the `BuildTargetContext`? 

Will make it less painful when we go to refactor this thing.